k6/src/k6/utils.ts

```
@@ -0,0 +1,93 @@
+    public static extendWithRandomBytes(bodyString: string): string {
+    const body = JSON.parse(bodyString);
+    const count = MaxPayloadBytes - bodyString.length - 1;
+    let randomContent = "";
+    for (let i = 0; i < count; i++) {
+        const idx = Math.floor(Math.random() * Alphabet.length);
+        randomContent += Alphabet.charAt(idx);
+    }
+    return JSON.stringify({ ...body, randomContent });
+    }
+
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)